connector-proxy-demo/connector_formsflow/commands/form_submission.py

```diff
@@ -0,0 +1,86 @@
+from http import HTTPStatus
+from typing import Dict
+from spiffworkflow_connector_command.command_interface import CommandErrorDict
+from spiffworkflow_connector_command.command_interface import ConnectorCommand
+from spiffworkflow_connector_command.command_interface import ConnectorProxyResponseDict
+from spiffworkflow_connector_command.command_interface import CommandResponseDict
+
+from connector_formsflow.services.external.formio_service import FormIOService
+from connector_formsflow.services.external.spiff_api_service import SpiffBackendService
+from connector_formsflow.services.utils.type_defs import TaskData, UpdateTaskDataPayload
+
+
+class FormSubmission(ConnectorCommand):
+
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, config, task_data: TaskData):
+        error: CommandErrorDict | None = None
+        logs = []
+        try:
+            logs.append("Creating submission")
+            logs.append("Fetching last submission")
+            formio_service = FormIOService()
+            form_url = task_data["data"]["formUrl"]  # type: ignore
+            submission = formio_service.fetch_submission(form_url)
+            logs.append("Creating revision")
+            submission_url = formio_service.get_submission_url(form_url)
+
+            logs.append("Calling FormIO api")
+            revision_id = formio_service.create_revision(submission_url, submission)
+            logs.append(f"{revision_id = }")
+            logs.append("Updating task data")
+            payload = self._prepare_payload(task_data, revision_id)
+            response_json, status_code = SpiffBackendService.update_task_data(
+                task_data["task_guid"], payload  # type: ignore
+            )
+            logs.append(f"Response from api {response_json}")
+            if status_code != HTTPStatus.OK:
+                error = {
+                    "error_code": "API_CALL_FAILED",
+                    "message": f"Spiff API call failed. {response_json = }",
```

Comment:
` = `after response_json?

Reply:
It is an F-string feature. The output string will be formatted like `Spiff API call failed. response_json = {response json value}`